bridgebot/bidding.py

```diff
@@ -24,23 +126,50 @@ def __init__(self, first_bidder):
             Players.WEST: []
         }
 
-    def __ns_first_bid(self, strain):
-        return
-        self.ns_first_bid = {
-            Strains.CLUBS: None,
-            Strains.DIAMONDS: None,
-            Strains.HEARTS: None,
-            Strains.SPADES: None,
-            Strains.NT: None
+        # Not necessary but makes it more human readable
+        if dealer == Players.EAST:
+            self.record[Players.NORTH].append("-")
+        elif dealer == Players.SOUTH:
+            self.record[Players.NORTH].append("-")
+            self.record[Players.EAST].append("-")
+        elif dealer == Players.WEST:
+            self.record[Players.NORTH].append("-")
+            self.record[Players.EAST].append("-")
+            self.record[Players.SOUTH].append("-")
+
+        self.first_bids = {
+            Team.NS: {
+                Strains.CLUBS: None,
+                Strains.DIAMONDS: None,
+                Strains.HEARTS: None,
+                Strains.SPADES: None,
+                Strains.NT: None
+            },
+            Team.EW: {
+                Strains.CLUBS: None,
+                Strains.DIAMONDS: None,
+                Strains.HEARTS: None,
+                Strains.SPADES: None,
+                Strains.NT: None
+            }
         }
 
-        self.ew_first_bid = {
-            Strains.CLUBS: None,
-            Strains.DIAMONDS: None,
-            Strains.HEARTS: None,
-            Strains.SPADES: None,
-            Strains.NT: None
-        }
+    def try_to_set_first_bid(self, player, strain):
```

Comment:
See lines 228-229. It considerably simplifies the logic that was formerly in `make_bid`